integration/kubernetes/helm-chart/alluxio/templates/config/alluxio-conf.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  annotations:
+    "helm.sh/hook": "pre-install"
+    "helm.sh/hook-delete-policy": before-hook-creation
+  name: {{ template "alluxio.fullname" . }}-config
+  labels:
+    name: {{ template "alluxio.fullname" . }}-config
+    app: {{ template "alluxio.name" . }}
+    chart: {{ template "alluxio.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+data:
```

Comment:
we seem to have lost the recent changes by @jiacheliu3 for supporting the embedded journal configuration with helm charts (https://github.com/Alluxio/alluxio/blob/master/integration/kubernetes/helm/alluxio/templates/alluxio-configMap.yaml#L14). can you please merge those in?

Reply:
Can we do this in another PR?

Reply:
sgtm